build/ansible/pmm2/post-build-actions.yml

```diff
@@ -44,6 +44,28 @@
     - name: Enable repo                | Enable release repo for pmm2-client
       command: percona-release enable {{ pmm_client_repos_final }}
 
+    - name: Detect Docker
```

Comment:
I don't think we need to "detect" docker, it is already done and known if you look at https://github.com/percona/pmm/pull/2308/files#diff-ca10aefe274646dba1a8c6cf00ce819d94ba0f6e40b3050eeffd9d54643f9046R14

Reply:
We can continue using `when: ansible_virtualization_type == "docker"`, since the contents of that file invariably depend on this condition. This will simplify the maintenance.

Reply:
Fixed